applications/doodle/src/module/cf_sms_user.erl

```diff
@@ -25,7 +25,10 @@
 -spec handle(wh_json:object(), whapps_call:call()) -> 'ok'.
 handle(Data, Call1) ->
     UserId = wh_json:get_ne_value(<<"id">>, Data),
-    Call = doodle_util:set_callee_id(UserId, Call1),
+    Funs = [{fun doodle_util:set_callee_id/2, UserId}
```

Comment:
Why are you doing this?
Will need to test, but this might be an issue.  What if the callee_id has already been set otherwise?